lib/kubeclient/common.rb

```diff
@@ -194,8 +194,8 @@ def define_entity_methods
           get_entity(klass, entity.resource_name, name, ns, opts)
         end
 
-        define_singleton_method("delete_#{entity.method_names[0]}") do |name, namespace = nil|
-          delete_entity(entity.resource_name, name, namespace)
+        define_singleton_method("delete_#{entity.method_names[0]}") do |name, ns = nil, opts = {}|
```

Comment:
@kamerblauwlicht we'd like to keep `namespace` for consistency. You can split the line to make rubocop happy. cc @NickLaMuro

Reply:
understood, but I can't split the line when using a block. can you show me an example to split @simon3z

Reply:
@kamerblauwlicht check this https://github.com/abonas/kubeclient/pull/271#pullrequestreview-68545556
Since you're at it could you change also the other `ns` instance that was recently introduced in the code? :blush:

Reply:
hahah I will @simon3z  ;-)